agrirouter-api-java-api/src/main/kotlin/com/dke/data/agrirouter/api/service/parameters/SetCapabilitiesParameters.kt

```diff
@@ -28,6 +28,9 @@ class SetCapabilitiesParameters : AbstractParameterBase(), ParameterValidation {
 
     lateinit var capabilitiesParameters: List<CapabilityParameters>
 
+    var enablePushNotifications: Capabilities.CapabilitySpecification.PushNotification = Capabilities.CapabilitySpecification.PushNotification.DISABLED
```

Comment:
Are we shure, that this will be the default behavior? I know, that leaving this field as`lateinit` will cause a breaking change, but isn't this a choice the user should make?

Reply:
Exactly that was my thought, I want to avoid a breaking change and this way we stay compatible. From my point of view, any developer can feel free to change this behavior, but if we don't set a standard value, we have a breaking change, which I wanted to avoid. Is there any advantage in having a breaking change?

Reply:
Because we do not have any release version yet - a breaking change would be not a big deal. We'll use the default value, because it fits most of the developers.